providers/implementations/rands/crngt.c

```diff
@@ -125,7 +118,18 @@ size_t prov_crngt_get_entropy(PROV_DRBG *drbg,
     if (crngt_glob == NULL)
         return 0;
 
-    if ((pool = rand_pool_new(entropy, 1, min_len, max_len)) == NULL)
+    /*
+     * Calculate how many bytes of seed material we require, rounded up
+     * to the nearest byte.
+     */
+    bytes_needed = (entropy + 7) / 8;
+    bytes_needed *= 1;                      /* scale for quality of source */
```

Comment:
I don't understand this line.

Reply:
We don't currently support entropy sources that provide less than full entropy.  Once we do support such devices, we'll need a scaling step and it needs to go here.  I.e. this is a placeholder for such a step.  I can replace this line with a better comment if that would help.

Reply:
Added a comment and removed this line.